engine/apps/alerts/migrations/0015_alertgroup_response_time.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-03-20 14:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0014_alertreceivechannel_restricted_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Currently I don't think we can't handle adding this to the table, migration takes too long in production.  Is there another way we can store it?